.changeset/sharp-fans-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/oauth-providers': minor
```

Comment:
I think it will be `patch` release because this PR is `fix`.

How do you think?

Reply:
Right! My bad. I'll change it now